rest_framework/pagination.py

```diff
@@ -534,7 +540,7 @@ def get_next_link(self):
         if not self.has_next:
             return None
 
-        if self.cursor and self.cursor.reverse and self.cursor.offset != 0:
+        if self.cursor and self.cursor.reverse and self.cursor.offset != 0 and self.page:
```

Comment:
This change seems unrelated to changing `OFFSET_CUTOFF` to instead be a class attribute?